shield_with_impacts.gdshader

```diff
@@ -1,42 +1,69 @@
 shader_type spatial;
 
-render_mode unshaded;
+render_mode unshaded, cull_disabled;
```

Comment:
Feel free to remove `cull_disabled` here.

I added it because I wanted to be able to see impacts on the front of the shield from behind it. Without this, the back face of the front of the shield gets culled and you can't see the impact happening.

Of course, this also makes the shield "extra thick" since you can see the back and front of it at the same time, so it's a bit of a trade-off.